pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -112,6 +112,7 @@ def __init__(self, filepath: Optional[str] = None, monitor: str = 'val_loss', ve
             if os.path.isdir(filepath):
                 self.dirpath, self.filename = filepath, '{epoch}'
             else:
+                filepath = os.path.realpath(filepath)
```

Comment:
what about abspath?

Reply:
The good thing about `realpath` is that also calls `abspath` in [itself](https://github.com/python/cpython/blob/3.7/Lib/posixpath.py#L391) 😀